packages/e2e-tests/specs/widgets/adding-widgets.test.js

```diff
@@ -204,8 +204,13 @@ describe( 'Widgets screen', () => {
 		);
 		await paragraphBlock.click();
 
-		const firstParagraphBlock = await firstWidgetArea.$(
-			'[data-block][data-type="core/paragraph"][aria-label^="Empty block"]'
+		const firstParagraphBlock = await page.waitForFunction(
+			( widgetArea ) =>
+				widgetArea.querySelector(
+					'[data-block][data-type="core/paragraph"][aria-label^="Empty block"]'
+				),
+			{},
+			firstWidgetArea
```

Comment:
Oh, I guess there's no elementHandle.waitForSelector, so this is the only option.